gen3/bin/kube-setup-hatchery.sh

```diff
@@ -20,13 +20,64 @@ gen3 jupyter j-namespace setup
 #
 (g3k_kv_filter ${GEN3_HOME}/kube/services/hatchery/serviceaccount.yaml BINDING_ONE "name: hatchery-binding1-$namespace" BINDING_TWO "name: hatchery-binding2-$namespace" CURRENT_NAMESPACE "namespace: $namespace" | g3kubectl apply -f -) || true
 
-
+# TODO: disable this cron-job after hatchery is updated to the dynamoDB version
```

Comment:
I think we can move this block of logic that sets up the `distribute-licenses` cronjob into the if block at https://github.com/uc-cdis/cloud-automation/pull/2441/files#diff-60cb659c9ec13c9256d2140bfb074068431ad6177cfe217baf6ee5210c125d04R74
This way we could maintain the backward compatibility but not creating this cronjob if the new config is in place
We will be eventually removing this part of logic soon from this script